client/scss/utilities/_root-scope-classes.scss

```diff
@@ -18,6 +30,19 @@
   margin-bottom: $spacing-unit * 2;
 }
 
+.l-margin-s {
+  margin-left: $spacing-unit * 2;
+}
+
+.r-margin-s {
+  margin-right: $spacing-unit * 2;
+}
+
+.h-center {
```

Comment:
I'm not familiar with the class structure conventions on this site, but it seems like all of these are 1to1 `classname -> property` mappings. Kind of like tachyons. Is there a reason for doing this instead of, for instance, styling `.pagination` with all those properties?